.travis.yml

```diff
@@ -45,6 +45,10 @@ matrix:
       env: PROJECT_DIR=examples/OpenCV TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon
     - os: linux
       env: PROJECT_DIR=examples/OpenCV TOOLCHAIN=android-ndk-r15c-api-21-armeabi-v7a-neon-clang-libcxx VERBOSE=0
+    - os: linux
+      env: PROJECT_DIR=examples/OpenCV-Qt TOOLCHAIN=gcc
+    - os: linux
+      env: PROJECT_DIR=examples/OpenCV-Qt TOOLCHAIN=gcc-pic
```

Comment:
There is no such toolchain tested with Qt. See https://travis-ci.org/ingenue/hunter/builds/280604163

Reply:
I guess we need some system libraries for Qt:
* https://github.com/ingenue/hunter/blob/c68630f7865f1fcf5adc58218392aa642205e780/.travis.yml#L22-L26

Reply:
> At this moment it is trying to build Qt with toolchain "gcc-pic", I guess I should just disable this test?

Yes